4.0/Makefile

```diff
@@ -0,0 +1,77 @@
+BUILDDIR=build
+TOOLSDIR=tools
+SOURCEDIR=en
+TEMPLATEDIR=templates
+SOURCES = $(SOURCEDIR)/*.md
+TARGETS=$(addprefix $(BUILDDIR)/,$(wildcard $(SOURCES)))
+TARGETNAME=OWASP Application Security Verification Standard 4.x
```

Comment:
Would the version number on this name be hand-updated for each release or could it be generated?